metaphor/notion/config.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+from pydantic.dataclasses import dataclass
+
+from metaphor.common.base_config import BaseConfig
+from metaphor.common.dataclass import ConnectorConfig
+
+
+@dataclass(config=ConnectorConfig)
+class NotionRunConfig(BaseConfig):
+    # notion integration authorization token
+    notion_api_tok: str
```

Comment:
```suggestion
    notion_api_token: str
```

Unless it's commonly referred as `notion_api_tok` elsewhere.